static/app/components/feedback/hydrateFeedbackRecord.tsx

```diff
@@ -6,40 +6,10 @@ import {
 export default function hydrateFeedbackRecord(
   apiResponse: RawFeedbackItemResponse
 ): HydratedFeedbackItem {
-  // const unorderedTags: HydratedFeedbackItem['tags'] = {
```

Comment:
I see dead code
![image](https://github.com/getsentry/sentry/assets/1633368/4f387528-4a9c-4719-8d66-6fc5875a05c2)

Is this going to be deleted before merging?

Reply:
yes this PR is deleting this code!